src/App.js

```diff
@@ -104,6 +104,9 @@ const App = () => {
       .catch((error) => console.log("logout errors: ", error))
   }
 
+  const updateReview = (editReview, id) => {
+
```

Comment:
While I usually would say to remove any console logs, until your fetch calls are implemented, I would suggest having a console log as a place holder showing the content being passed in as params.